models/issues/issue.go

```diff
@@ -110,12 +119,13 @@ type Issue struct {
 	Milestone        *Milestone             `xorm:"-"`
 	Project          *project_model.Project `xorm:"-"`
 	Priority         int
-	AssigneeID       int64            `xorm:"-"`
-	Assignee         *user_model.User `xorm:"-"`
-	IsClosed         bool             `xorm:"INDEX"`
-	IsRead           bool             `xorm:"-"`
-	IsPull           bool             `xorm:"INDEX"` // Indicates whether is a pull request or not.
-	PullRequest      *PullRequest     `xorm:"-"`
+	AssigneeID       int64             `xorm:"-"`
+	Assignee         *user_model.User  `xorm:"-"`
+	IsClosed         bool              `xorm:"INDEX"`
+	ClosedStatus     IssueClosedStatus `xorm:"NOT NULL DEFAULT 0"`
```

Comment:
Could rename to just `State` so it could be used for multiple open states as well in future.

Reply:
I would strongly suggest to use two words. One-word column would cause ambiguity easily, and sometimes when the single-word is a database keyword, it causes SQL failures when the quoting is forgotten.